Topic 1/src/point7/Music.java

```diff
@@ -0,0 +1,78 @@
+package point7;
+
+public class Music {
+	private String disk; // indicates which disk or album the song belongs
+	private String artist;
+	private String title;
+	private int duration;
+	private boolean state; // indicates whether this on or off
+	
+	public Music(String disk, String artist, String title, int duration){
+		this.setDisk(disk);
+		this.setArtist(artist);
+		this.setTitle(title);
+		this.setDuration(duration);
+		this.setState(false);
+	}
+
+	public boolean equals (Object o) {
+		Music d= (Music)o;
+		return ((d.getTitle().equals(this.getTitle())) && (d.getDisk().equals(this.getDisk())));
```

Comment:
[High] In case that you have the *title* or *disk* in <code>null</code>, then it will throw a *NullPointerException* error.
You should check those conditions